docs/support/managing-ckeditor-logo.md

```diff
@@ -0,0 +1,73 @@
+---
+category: licensing
+order: 30
+toc: false
+meta-title: CKEditor 5 support documentation
+meta-description: Learn how to get help and support and how to provide feedback.
+---
+
+# Managing the "Powered by CKEditor" logo
+
+## Why the "Powered by CKEditor" logo?
+
+Starting from version v38.0.0 onwards, all **open source installations** of CKEditor 5 (free users) carry a small “Powered by CKEditor” watermark in the bottom right corner of the editing area. The label links directly to the [CKEditor website](https://ckeditor.com/) and while it may be a little frustrating to some. This new branding approach is designed to make sure the entire community knows who is powering and modernizing their rich text editor.
+
+{@img assets/img/powered-by-ckeditor.png Placement of the "Powered by CKEditor" logo within the editor}
+
+It is also another way for us to gather feedback from users and continue to improve this major open source project that we have been supporting for more than 20 years. CKEditor has 40+ full time engineers and R&D professionals focused on its continual evolution, with 30K+ developers now using it on a daily basis.
+
+This change **does not affect customers with commercial licenses**, but please read on as certain actions need to be taken. You can reach out to our Technical Support team, using [this form](https://ckeditor.com/contact/), if you have any questions.
+
+## How to remove the "Powered by CKEditor" logo?
+
+In order to remove the logo, you need a commercial license for CKEditor 5. If you need one, please [contact us](https://ckeditor.com/) for a tailored offer. If you already have one, please read on.
+
+### Stand-alone features
```

Comment:
What's the purpose of this section? People who have those standalone features have this key configured already.

Reply:
Don't they need to update the key to the new one?

Reply:
In commit.